bootcamp/templates/navbar.html

```diff
@@ -0,0 +1,19 @@
+{% load static %}
+  <nav class="navbar" role="navigation">
+      <div class="navbar-header">
+        <div class="pull-left">
+          <a class="brand-logo" href="http://www.mit.edu"></a>
+          <div class="divider-large">&nbsp;</div>
+          <a class="bootcamp-logo" href="/"></a>
+          </div>
+        <div class="pull-right">
+          {% if authenticated %}
+          <a class="header-dashboard-link" href="/logout">
+            Sign Out
+          </a>
+          {% else %}
+          <a class="header-dashboard-link" href="/logout">Not logged in</a>
```

Comment:
Can you remove the link here? If the user isn't logged in we don't need a link here